plugins/plugin-bash-like/src/lib/cmds/git-diff.ts

```diff
@@ -17,20 +17,16 @@
 import * as Debug from 'debug'
 const debug = Debug('plugins/bash-like/cmds/git-diff')
 
-import * as path from 'path'
-
 import { split } from '@kui-shell/core/core/repl'
 import { isPopup } from '@kui-shell/core/webapp/cli'
 import Presentation from '@kui-shell/core/webapp/views/presentation'
 import { CommandRegistrar, IEvaluatorArgs } from '@kui-shell/core/models/command'
 
 import { handleNonZeroExitCode } from '../util/exec'
 import { asSidecarEntity } from '../util/sidecar-support'
-import { onbranch, injectCSS } from '../util/git-support'
+import { onbranch } from '../util/git-support'
 
 const doDiff = async ({ command, execOptions }: IEvaluatorArgs) => new Promise(async (resolve, reject) => {
-  const injector = injectCSS()
```

Comment:
This doesn't seem right. Remove this line will not load git-diff css. Maybe change `const injector = injectCSS()` to `injectCSS()`